server/public/model/link_metadata.go

```diff
@@ -89,7 +90,7 @@ func (o *LinkMetadata) PreSave() {
 }
 
 func (o *LinkMetadata) IsValid() *AppError {
-	if o.URL == "" {
+	if o.URL == "" || len(o.URL) > LinkMetadataMaxURLLength {
```

Comment:
We should have a separate error string for this case. `model.link_metadata.is_valid.url.app_error` just says `Link metadata URL must be set` which has no relation to max URL length.

Please create a separate error string, and mention the current length and the maximum allowed length by passing params to model.AppError.

Reply:
https://github.com/mattermost/mattermost/pull/29589/commits/7c41e021df0264f505d370a074bb187cbec69423

In which file should I create the error string?

Reply:
https://github.com/mattermost/mattermost/pull/29589/commits/0e7b07bd760089057e72af426d412aa89f81d39d

Done.

Reply:
> Please create a separate error string, and mention the current length and the maximum allowed length by passing params to model.AppError.

This is still not done. You need to add context to the error by adding parameterized strings like `{{maxLen}}` and pass the param value with a map. See for example: `model.outgoing_oauth_connection.is_valid.audience.error`.

Reply:
https://github.com/mattermost/mattermost/pull/29589/commits/866322ca192c54b05e1b85d5305c753e518ea95f